sunpy/tests/helpers.py

```diff
@@ -79,45 +79,25 @@ def figure_test(test_function):
     def test_simple_plot():
         plt.plot([0,1])
     """
-    @pytest.mark.figure
+    ft2_version = f"{mpl.ft2font.__freetype_version__.replace('.', '')}"
+    mpl_version = "dev" if "+" in mpl.__version__ else mpl.__version__.replace('.', '')
+    astropy_version = "dev" if "dev" in astropy.__version__ else astropy.__version__.replace('.', '')
+    hash_library_name = f"figure_hashes_mpl_{mpl_version}_ft_{ft2_version}_astropy_{astropy_version}.json"
+    hash_library_file = Path(__file__).parent / hash_library_name
+
+    @pytest.mark.figure  # Add for backwards compatibility
```

Comment:
Is the plan to leave this here forever?

Reply:
Maybe we don't need it at all. The reason to keep it was to keep `-m "figure"` working

Reply:
What would it be without this? `-m mpl-figure`?

Reply:
`-m mpl_image_compare` as done in tox.ini

Reply:
That isn't so bad. I feel like we could remove "figure"

Reply:
Yeah, I would say that there are probably so few people running the figure tests locally that changing this is fine.